clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -114,7 +114,7 @@
     private static final Logger log = LoggerFactory.getLogger(KafkaAdminClientTest.class);
 
     @Rule
-    final public Timeout globalTimeout = Timeout.millis(120000);
+    final public Timeout globalTimeout = Timeout.millis(120000000);
```

Comment:
Why did we increase this?

Reply:
Sorry, did this for debugging and forgot to remove.